src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig

```diff
@@ -36,8 +35,26 @@
         <div class="spinner spinner-primary"></div>
       </div>
 
-      {# Integrate the token that will be required in the combination update request #}
-      {{ form_row(combinationsForm, {'attr': {'data-combination-token': csrf_token(combinationItemForm.vars.name) }}) }}
+      <div id="combinations-list-form-container">
+        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Combination/combination_list_form.html.twig', {'combinationsForm': combinationsForm}) }}
+      </div>
+
+      <div id="combinations-list-footer">
+        <span>
+          {{ 'Edition mode'|trans({}, 'Admin.Actions') }}
```

Comment:
Can you tell me where the string appears, please? I couldn't find it on the mockups.

Reply:
It's above the regular page of combination in the mockup. I included a screenshot in the description of this PR so you can find it more easily.